README.md

```diff
@@ -1,15 +1,42 @@
-Bundle App
-Help people who don't travel often or are going somewhere new to feel less anxious about preparing and packing right for the trip. Our app helps with managing overall details of the trip and packing from one hub.
+![Bundle The Trip Readiness](https://lh3.googleusercontent.com/1KVsbCJ0Z6z7bUxsN6YgSwKtxI5adDgww0q2nP2Lf2i2Jc3Bzcz6mZ8Brpnf0x9yaH9eUPzX2vBD4CjNyZywyFeAssyvyahFW-elfycirHGFxAYVrWNbJAhrcqS9LYiglxg7c4kYJw=w1135-h709-no "Bundle Logo")
+# [**Bundle**](http://bundle.tips)
+### _A **mobile-first**, trip-readiness app_
+---
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Bundle relieves packing worries by showing you what to pack from one place. 
```

Comment:
Bundle relieves packing worries by showing travel goers what to pack from one place.